frontend/src/routes/Governance/discovered/ByCluster/ConfigByCluster.tsx

```diff
@@ -0,0 +1,41 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import { PageSection } from '@patternfly/react-core'
+import { DiscoverdPolicyItem } from '../useFetchPolicies'
+import { AcmEmptyState, AcmTable } from '../../../../ui-components'
+import { useTranslation } from '../../../../lib/acm-i18next'
+import { useRecoilValue, useSharedAtoms } from '../../../../shared-recoil'
+import { Box } from '@mui/material'
+import { ByClusterCols, DiscoveredViolationsCard, policyViolationSummary } from './common'
+
+export default function ConfigByCluster({ policies }: Readonly<{ policies: DiscoverdPolicyItem[] }>) {
```

Comment:
Could we just have one component that handles all policy kinds rather than one per kind? This makes adding new kinds easier and less code duplication.

Reply:
doable